src/nvim/lua/executor.c

```diff
@@ -2217,3 +2217,38 @@ char *nlua_read_secure(const char *path)
 
   return buf;
 }
+
+char *nlua_trust(const char *action, int bufnr)
```

Comment:
I split this back into two separate functions. `nlua_trust` for the lua api wrapper, and `ex_lua` in `ex_cmds.c` for the command.